loops-and-arrays.js

```diff
@@ -4,35 +4,48 @@
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 namen zou bevatten!
 // ==========================================
 
-const names = ["Henk", "Piet", "Fred", "Joop"];
-
 // Verwachtte uitkomsten:
+
 // Vóór het script zie je de originele waardes:
 // console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
+const names = ["Henk", "Piet", "Fred", "Joop"];
 
+console.log(names);
 
+for (let i = 0; i < names.length; i++ ) {
+    names[i]=`${names[i]}je`
+}
 
+console.log(names);
 
 // ==========================================
 // Opdracht 2
 // Schrijf een script dat ieder EVEN getal in onderstaande array met 2 vermenigvuldigd, en ieder ONEVEN getal met 3
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
-const numbers = [2, 4, 5, 29, 38];
-
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(numbers) geeft: [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 4, 8, 15, 87, 76 ];
 
+const numbers = [2, 4, 5, 29, 38];
 
+console.log(numbers);
 
+for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+        numbers[i] = 2 * numbers[i];
+    } else {
+        numbers[i] = 3 * numbers[i];
```

Comment:
Hoi Patrick, ik weet het niet zeker maar is het niet netter om numbers[i] * 2; te schrijven?